libfwupdplugin/fu-common.h

```diff
@@ -89,7 +93,11 @@ typedef enum {
 	FU_PATH_KIND_SYSFSDIR_SECURITY,
 	FU_PATH_KIND_ACPI_TABLES,
 	FU_PATH_KIND_LOCKDIR,
+<<<<<<< HEAD
 	FU_PATH_KIND_SYSFSDIR_FW_ATTRIB,
+=======
+	FU_PATH_KIND_LOCKFILE,
+>>>>>>> 608f1be3 (CHROMIUM: inhibit system suspend during update)
```

Comment:
looks like a bad merge/rebase

Reply:
Yeah, I had @binaryNewts force push this back to a commit from earlier, and submit their Inhibit Suspend change in  separate PR, #3489. That one is going to be dependent on the initial skeleton, and I'll help them rebase that work once the skeleton is merged.